src/translators/sql_translator.py

```diff
@@ -805,8 +812,10 @@ def query(self,
             try:
                 self.cursor.execute(op)
             except Exception as e:
+                # TODO due to this except in case of sql errors,
+                # all goes fine, and users gets 404 as result
                 # Reason 1: fiware_service_path column in legacy dbs.
-                logging.debug("{}".format(e))
+                logging.error("{}".format(e))
```

Comment:
for now we just promoted the log to error, but i think it would be correct just to raise the exception to the api call level

Reply:
yea, we should. Also, I've got no clue what that comment (reason 1) actually means. Does it ring a bell to you?

Reply:
i think it was something introduced by tomas, I don't have clue about it. a simple approach could be raise the exception, and see if tests work fine or breaks. if work fine, we go that way now, if not we keep the log, and later fix to introduce proper error handling

Reply:
tested, things break because it seems that after you drop tables (so they are not there), having still content in the metadata table (probably due to needed refresh time), ql attempts to query them and bombs.

Reply:
cool, thanks for this!! let's keep the log for now then. I've opened an issue about it just in case we want to revisit this scenario in the future: #372.